tests/basic.rs

```diff
@@ -344,8 +357,7 @@ fn create_storages_all() {
 }
 
 #[test]
-#[should_panic(expected = "Cannot create storage at \\\"/foo/bar\\\" \
-                           because a stream already exists there")]
+#[should_panic]
```

Comment:
Yes, because the tests for Windows are falling

Reply:
Oh, I see; I guess that on Windows the path in the error message looks like "foo\\bar" instead of "foo/bar"?

Perhaps we could shorten the expected substring to e.g. `"Cannot create storage"`, so there's still some indication that we're getting the panic we expect?

Reply:
Done